Npgmq.Test/NpgmqClientTest.cs

```diff
@@ -872,6 +872,37 @@ public async Task SetVtAsync_should_change_vt_for_a_message()
         Assert.Equal(msgId, message2.MsgId);
     }
 
+    [SkippableFact]
+    public async Task SetVtBatchAsync_should_change_vt_for_multiple_messages()
+    {
+        // Arrange
+        Skip.If(!_pgmqSupportsPopQty, "This test requires pgmq 1.8.0 or later");
```

Comment:
What is `_pgmqSupportsPopQty`?

Why did you try to do this differently than the other tests that skip based on pgmq version?